.github/workflows/scan-docker-images.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: "Scan Docker images"
+
+on:
+  schedule:
+    - cron: "15 1 * * 0"
```

Comment:
I'm sorely disapointed that github doesn't just support `@weekly` so they can load balance.

Reply:
🤷 I felt the same way, but I guess I wouldn't care to deal with the ramifications of GitHub-wide managed `weekly` load balancing of tasks. If I were on the GHA team, I'd prolly just be salty having to implement that instead of letting users just write their own unique schedules.